example.env.local

```diff
@@ -0,0 +1,11 @@
+// Populate this file from the Web App & Realtime Database 
+// you create at https://console.firebase.google.com
+// See the README.md for setup details.
```

Comment:
examples of what this might look like could help but since we' re the only two that have ever opened this app up... :P